src/provisioning/Provisioner.ts

```diff
@@ -132,12 +132,18 @@ export class Provisioner {
 
         // Deal with CORS (temporarily for s-web)
```

Comment:
This comment which once described the entire purpose of this handler has now become outdated.